partner_tag_smart_assignation/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
```

Comment:
`# Copyright (C) 2019 Compassion CH (http://www.compassion.ch)`
`# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).`
@ecino Change Copyright like this